CHANGELOG.md

```diff
@@ -1,5 +1,168 @@
 # Release Notes
 
+## Version 1.5.0-beta.0 - 2019-10-15
+
+* Replace registry-creds addon ReplicationController with Deployment [#5586](https://github.com/kubernetes/minikube/pull/5586)
+* Performance and security enhancement for ingress-dns addon [#5614](https://github.com/kubernetes/minikube/pull/5614)
+* Add addons flag to 'minikube start' in order to enable specified addons [#5543](https://github.com/kubernetes/minikube/pull/5543)
+* Disable go modules for containerd, for newer go [#5608](https://github.com/kubernetes/minikube/pull/5608)
+* Upgrade Buildroot to 2019.02 and VirtualBox to 5.2 [#5609](https://github.com/kubernetes/minikube/pull/5609)
+* gsutil: Remove trailing slashes, rsync instead of cp [#5620](https://github.com/kubernetes/minikube/pull/5620)
```

Comment:
The changelog should contain user visible changes only, and specifically exclude any changes which are not shipped as part of the binary or ISO. Specifically, the changelog should omit:

- CI changes
- Documentation-only changes
- Test-only changes

It doesn't have to be exact, but filter as best as you can for them. The fewer entries the better, IMHO. 

(We should open a bug to automatically filtering out changes by file extension in the future)

Reply:
Done, remaining PRs should be user facing. Let me know if I missed any I should delete.